mock_services/mock_data/index.js

```diff
@@ -4,10 +4,12 @@ const {
 } = require("mongo-seeding");
 
 const config = {
-  database: `${process.env.MONGODB_URI}/${process.env.MONGODB_DB_NAME}?authSource=admin`,
-  dropDatabase: true, // Drop database before import
+  database: `mongodb://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PWD}@localhost:27017/${process.env.MONGODB_DB_NAME}?authSource=admin`,
+  dropDatabase: false, // Drop database before import
```

Comment:
I'm not sure if this is a good idea tbh.
As is you could populate random data into your "prod" db. I think leaving it as was before would be better, but I also see how it could be dangerous cause then you're dropping a whole db vs. "infecting" it with random data. lmk what you think

Reply:
Agreed, I reverted and instead added user input (y/n) before the operation continues so that the user confirms they're okay with dropping the DB.